From 7db15a99f676bea3aebe4b5303f17862295b3ef3 Mon Sep 17 00:00:00 2001
From: kindle4jerry <kindle4jerry@gmail.com>
Date: Mon, 17 Apr 2023 21:32:06 +0800
Subject: [PATCH 1/4] build dont handle apns conf

---
 target/product/full_base_telephony.mk | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/target/product/full_base_telephony.mk b/target/product/full_base_telephony.mk
index f4f5787..7c68280 100644
--- a/target/product/full_base_telephony.mk
+++ b/target/product/full_base_telephony.mk
@@ -26,10 +26,6 @@ PRODUCT_VENDOR_PROPERTIES := \
 PRODUCT_COPY_FILES := \
     frameworks/native/data/etc/handheld_core_hardware.xml:$(TARGET_COPY_OUT_VENDOR)/etc/permissions/handheld_core_hardware.xml
 
-ifeq ($(EXTHM_BUILD),)
-PRODUCT_COPY_FILES += \
-    device/sample/etc/apns-full-conf.xml:system/etc/apns-conf.xml
-endif
 
 $(call inherit-product, $(SRC_TARGET_DIR)/product/aosp_base.mk)
 $(call inherit-product, $(SRC_TARGET_DIR)/product/telephony.mk)
-- 
2.25.1

