From 43bef3db21b02959302ba80443f76f1cf0bdf8b9 Mon Sep 17 00:00:00 2001
From: kindle4jerry <kindle4jerry@gmail.com>
Date: Mon, 17 Apr 2023 21:16:40 +0800
Subject: [PATCH] SubscriptionController: Do not override default calling

---
 .../telephony/SubscriptionController.java     | 19 ++++++++++++++++---
 1 file changed, 16 insertions(+), 3 deletions(-)

diff --git a/src/java/com/android/internal/telephony/SubscriptionController.java b/src/java/com/android/internal/telephony/SubscriptionController.java
index 82799be..1c614b9 100644
--- a/src/java/com/android/internal/telephony/SubscriptionController.java
+++ b/src/java/com/android/internal/telephony/SubscriptionController.java
@@ -2855,9 +2855,22 @@ public class SubscriptionController extends ISub.Stub {
                         subId);
 
         TelecomManager telecomManager = mContext.getSystemService(TelecomManager.class);
-
-        telecomManager.setUserSelectedOutgoingPhoneAccount(newHandle);
-        logd("[setDefaultVoiceSubId] requesting change to phoneAccountHandle=" + newHandle);
+        PhoneAccountHandle currentHandle = telecomManager.getUserSelectedOutgoingPhoneAccount();
+        logd("[setDefaultVoiceSubId] current phoneAccountHandle=" + currentHandle);
+
+        String currentPackageName =
+            currentHandle == null ? null : currentHandle.getComponentName().getPackageName();
+        boolean currentIsSim = "com.android.phone".equals(currentPackageName);
+        // Do not override user selected outgoing calling account
+        // if the user has selected a third-party app as default
+        boolean shouldKeepOutgoingAccount = currentHandle != null && !currentIsSim;
+
+        if (!Objects.equals(currentHandle, newHandle) && !shouldKeepOutgoingAccount) {
+            telecomManager.setUserSelectedOutgoingPhoneAccount(newHandle);
+            logd("[setDefaultVoiceSubId] change to phoneAccountHandle=" + newHandle);
+        } else {
+            logd("[setDefaultVoiceSubId] default phoneAccountHandle not changed.");
+        }
 
         if (previousDefaultSub != getDefaultSubId()) {
             sendDefaultChangedBroadcast(getDefaultSubId());
-- 
2.25.1

